lib/src/hooks.dart

```diff
@@ -64,7 +64,7 @@ List<String> getHooks(String hookRootPath) {
   var hooks = <String>[];
 
   if (exists(hookRootPath)) {
-    hooks = find('*.dart', workingDirectory: hookRootPath).toList();
+    hooks = find('*', workingDirectory: hookRootPath).toList();
 
```

Comment:
I think we should check that they are executable.

use global dcli function isExecutable

https://pub.dev/documentation/dcli/latest/dcli/isExecutable.html

Reply:
This is done in the calling function.